docs/source/tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Tutorial
+
+One of the main things that conda apart from other python packaging tool solutions is its ability to work with other languages and toolchains.  Conda
+is happy to mix Python with packages from R, Java, C, etc. This tutorial explores using **conda-project** to create a [Shiny](https://shiny.rstudio.com/)
+app using both **Jupyter notebook** on the Python side, and **shiny** on the R side.  After completing this tutorial, you will be able to:
+
+ * Understand the relationship between conda-project commands and their corresponding actions.
+ * Add a custom command to a project.
+ * Use the `run` action to launch external processes.
+
+If `conda-project` is not yet installed and started Project, follow the [Installation Instructions](index).
+
+## Create a new project
+
+This section of the tutorial explores the `create` and `activate` subcommands of conda-project.
+
+1. Open a Command Prompt or terminal window.
+
+2. Initialize the project in a new directory:
+   ```shell
+   conda-project create -n learn-cp --directory cp-tutorial --platform osx-arm64,linux-64 -c defaults -c conda-forge \
+       python=3.10 notebook r-essentials r-plotly r-shiny
+   ```
+
+   At this point, **conda-project** will create a new directory, and write environment, lock, and project files to the new
+   directory. This example passes the `osx-arm64` and `linux-64` platform architectures. This has the effect of speeding up
```

Comment:
How slow was it with the default platform list? I wonder if recommending libmamba would be useful? You could say that if `conda-libmamba-solver` is also installed you can pass `--conda-configs solver=libmamba`

Reply:
Running into issues with my setup coz of the RStudio conflict.  Setting a pristine env to try this.

Reply:
Installing with the default platform list with the default channels takes 4-ish minutes.  Also, I was having a little trouble getting this running in a pristine docker env with no R on-board.